torch/_dynamo/convert_frame.py

```diff
@@ -25,7 +25,7 @@
 from typing import Any, Callable, Dict, List, Optional, Set, Tuple, TypeVar, Union
 from typing_extensions import ParamSpec
 from weakref import ReferenceType
-
+import triton
```

Comment:
Sometimes triton isn't available actually. You can grep around to see how we import it elsewhere, But also: probably we can put the population of these fields in a common place: https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/utils.py#L965

Reply:
done